lib/ui/home/home_screen.dart

```diff
@@ -2,12 +2,17 @@ import 'package:flutter/material.dart';
 import 'package:flutter_riverpod/flutter_riverpod.dart';
 import 'package:survey_flutter_ic/extension/date_extension.dart';
 import 'package:survey_flutter_ic/extension/toast_extension.dart';
+import 'package:survey_flutter_ic/model/survey_model.dart';
+import 'package:survey_flutter_ic/theme/dimens.dart';
 import 'package:survey_flutter_ic/ui/home/home_header.dart';
 import 'package:survey_flutter_ic/ui/home/home_view_model.dart';
 import 'package:survey_flutter_ic/ui/home/home_view_state.dart';
+import 'package:survey_flutter_ic/ui/surveys/survey_view.dart';
+import 'package:survey_flutter_ic/widget/survey_shimmer_loading.dart';
 
 final _loadingStateProvider = StateProvider.autoDispose<bool>((_) => false);
 final _profileAvatarProvider = StateProvider.autoDispose<String>((_) => '');
```

Comment:
@manh-t Makes sense. I'd prefer to have `SurveyUiModel` as well. Can I refactor on the integration PR? It will avoid conflicts and rebase time since I am working on integration task 😅

Reply:
@hoangnguyen92dn 🙆‍♂️